lib/fuelsdk/soap.rb

```diff
@@ -376,12 +409,7 @@ def normalize_properties_for_cud object_type, properties
       properties = Array.wrap(properties)
       raise 'Object properties must be a Hash' unless properties.first.kind_of? Hash
 
-      if is_a_dataextension? object_type
-        format_dataextension_cud_properties properties
-      else
-        format_object_cud_properties object_type, properties
-      end
-
+      format_object_cud_properties object_type, properties
```

Comment:
It basically wasn't working. It was changing the format of the request in a way that made the SOAP envelope invalid. I'm not sure how/if this formatting ever worked.